### DISCUSSION TITLE
Need some help with a 4k Grabber 1/4  of the image is capture.d.

### DISCUSSION THREAD

**Discussion by wou5er** [awawa-dev/HyperHDR#110] [Q&A]
I've got [this](https://nl.aliexpress.com/item/4001294726740.html?spm=a2g0s.9042311.0.0.48684c4dCrBozX) Capture card... it supports 4k HDR etc... (U3 version)

But in HyperHDR I have an issue, only 1/4 of the screen is captured.. and shown in the Live Video. This results in a not so very ambiant, led and image projection.

I also have a max resolution of 720x480. on the pi. On linux Mint this doesn't ocure. 

Any suggestions how to fix this?

**Comment by awawa-dev** [comment]
I have this grabber unfortunately: for incoming 1080p it only allows 1080p (1920x1080) capturing and it's OK...but no 720x480 or whatever lower resolution. Probably that's why it's cropping the image in your case.
![u32](https://user-images.githubusercontent.com/69086569/131228241-72b1f04c-f440-4ee8-99e1-4fcb5428ed45.jpg)

![u3](https://user-images.githubusercontent.com/69086569/131228151-7d432c9a-eb2f-47ef-9edb-0fe481547445.png)

**Reply by awawa-dev** [reply]
BTW it doesnt support HDR

**Reply by wou5er** [reply]
After some doing I used a fresh minimal raspbian install instead of Hyperbian. Now overything works and to it does support 4k HDR or Dolby Vision on my xbox one + lg oled b8, using a hdmi matrix

**Comment by awawa-dev** [comment]
It doesn't have even HDR pass-through, not to mention HDR processing. 
https://blog.sreb.me/2021/03/03/rullz-navy-u3-review/

**Reply by wou5er** [reply]
I'm having a different version... I don't have the "Loop" version, but "Audio" 
Have to set the matrix to 4k30 but now got HDR working.

**Reply by awawa-dev** [reply]
It's the same stuff inside: I've got optical in the review it's analog audio, none works with HDR...if you want HDR then you have for example Ezcap 320 that can process HDR for 100$, and Ezcap 269 that can passthrough HDR for $70 without processing it ...not that stuff for $35. Do not mislead others users.
